db/schema.rb

```diff
@@ -11,15 +11,15 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150530010900) do
+ActiveRecord::Schema.define(version: 20150529162928) do
```

Comment:
the migration 20150530010900 doesn't exist, probably was created on a branch and flattened into a single migration